PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
+
+[//]: # (Put Issue # here, if applicable)
+Fixes #______
```

Comment:
Cool! Did not know that

On Sat, Mar 19, 2016 at 7:05 PM Alys <notifications@github.com> wrote:

> In PULL_REQUEST_TEMPLATE.md
> <https://github.com/HabitRPG/habitrpg/pull/6928#discussion_r56757176>:
>
> > @@ -0,0 +1,13 @@
> > +[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
> > +
> > +[//]: # (Put Issue # here, if applicable)
> > +Fixes #______
>
> No, just the URL, not the number at all. Either will cause a link to be
> made to the issue, and when the PR is saved, the URL will look like the
> number.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/6928/files/cf8b232d9498abed41c6d5b98dbb26ce4bf68f52#r56757176>
>